components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```
@@ -1111,6 +1115,89 @@
 
+def start_retention_cleaner(
+    instance_id: str,
+    clp_config: CLPConfig,
+    container_clp_config: CLPConfig,
+    mounts: CLPDockerMounts,
+):
+    component_name = RETENTION_CLEANER_COMPONENT_NAME
+
+    retention_cleaner_required = is_retention_cleaner_required(clp_config)
+    if not retention_cleaner_required:
+        logger.info(f"No retention period is configured, skipping {component_name}...")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)